headless/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM fedora:latest AS builder
+
+ARG TARGETARCH
+
+RUN yum install gcc glibc-devel zlib-devel libstdc++-static curl git -y
+RUN yum install zlib-devel zlib-static -y
+RUN yum install glibc-static -y
+
+# Install Enterprise GraalVM and Dependencies (needed to compile native image with JNI invocations)
+RUN cd / && mkdir graal
+
+COPY resources/graalvm-ee-java19-${TARGETARCH}.tar.gz /graal/graal.tar
+COPY resources/graal-config /graal/config
```

Comment:
Seems like these 2 files are missing from your contrib. Especially `graalvm-ee`. Could you download it in `Dockerfile` ?
And, is there any "licence"-ish need to be bought or paid when using "enterprise" version of GraalVM?